apps/utils/draft_pull_request.py

```diff
@@ -199,11 +193,6 @@ def cli(
     log.info("Creating an empty commit.")
     repo.git.commit("--allow-empty", "-m", title or f"Start a new staging server for branch '{new_branch}'")
 
-    if step_update:
-        log.info("Committing all files.")
```

Comment:
What if we simply print the list of files to be commited, and ask for confirmation, before adding and committing?

Reply:
If there are snapshot files in the list, the printed message should mention that snapshot scripts would need to be executed prior to committing (or otherwise expect CI to fail, as [suggested](https://github.com/owid/etl/pull/3205#discussion_r1732534404) by @lucasrodes).